package.json

```diff
@@ -1,4 +1,25 @@
 {
+  "name": "pwr2",
+  "description": "Docker image based on Ubuntu providing a base setup for a Phoenix+Webpack+Relay+React project, with some sugar and conventions to develop and build your own web applications.",
```

Comment:
Please add _"**Javascript dependencies for pwr2-docker, a** Docker image based on ...."_ in the description.